prime-router/examples/data-lineage/example-setup.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+
+echo
+echo "Creates the stored procedures that do lineage queries."
+docker exec --user 999:999 \
+  -i $(docker ps -f name=db_pgsql -q) \
+  psql prime_data_hub prime \
+    < ./lineage-report-stored-procedures.sql
+echo
```

Comment:
This step won't be needed any more - the stored process are going into the db permanently, once jim/lineage3 is merged.

Reply:
Merged in from master. Confirmed the stored procedures are in the clean DB.